syncer/syncer.go

```diff
@@ -209,10 +211,24 @@ func (s *syncer) BulkSync(newBlockCallback func(*types.Block) bool) error {
 	return nil
 }
 
+// makeSkipList initializes list of peers to skip
+func (s *syncer) makeSkipList() map[peer.ID]bool {
+	skipList := make(map[peer.ID]bool)
+
+	s.peerMap.Range(func(key, value interface{}) bool {
+		peer, _ := value.(*NoForkPeer)
+		skipList[peer.ID] = s.network.ShouldIgnoreSyncToPeer(peer.ID)
```

Comment:
Is it possible to rewrite this code by calling a method to given peer is bootnode or not?